### ISSUE TITLE
[PC ENGINE CD] Linux support seemingly broken

### ISSUE THREAD

**Issue by powerpuffboysz** [higan-emu/ares#1071]
i'm running version 132 on manjaro and when i attempt to load any pc engine cd games i either get stuck at the bios "just a moment screen" or crash to a black screen. this seems to be a linux only bug as the same bios and game images i'm attempting to use work just fine on the windows version both natively in windows as well as in wine.

**Comment by HeroponRikiBestest** [comment]
Unable to recreate; using Linux Mint 21, just pulled and compiled right now, and PC engine CD games load properly. 

![image](https://user-images.githubusercontent.com/50224630/230525381-cec5ed5c-4255-483e-a463-6510112883df.png)
![image](https://user-images.githubusercontent.com/50224630/230525446-fa886062-b921-408a-8309-170f40adf3d2.png)

**Comment by powerpuffboysz** [comment]
Might it be a flatpak issue? I did not compile from source.

**Comment by powerpuffboysz** [comment]
> Unable to recreate; using Linux Mint 21, just pulled and compiled right now, and PC engine CD games load properly.
> 
so i tried a few more games like valis III and gate of thunder/bonk's adventure/bon's revenge and they worked fine. however i still can't get either an unpatched or english patched rondo of blood to boot on linux while it does work fine in windows so i still think its a linux bug

**Comment by HeroponRikiBestest** [comment]
I had no issues getting Rondo of Blood to boot properly. I saw you mention here that you were using the flatpak; it's likely an issue with that rather than linux.
![image](https://user-images.githubusercontent.com/50224630/230702391-e4e86129-a278-4c64-bb38-756cc979521f.png)

**Comment by powerpuffboysz** [comment]
who maintains the flatpak so i can attempt to ask them about it?

**Comment by invertego** [comment]
I was able to load Rondo of Blood using the flatpak build of ares v132 in a VM running Linux Mint 21.1.

However, following a hunch I discovered that this game really doesn't like improperly formatted backup RAM. I was able to repro the issue on both Linux and Windows by deleting the PCE backup ram, loading a non-CD game, then loading Rondo of Blood.

Try formatting the PCE backup ram from the BIOS or deleting it outright before you load Rondo of Blood. You'll want to back it up first if you actually have made in-game saves in something. To delete it outright (assuming you installed ares via flatpak):
`rm ~/.var/app/dev.ares.ares/data/ares/Systems/PC\ Engine.sys/backup.ram`
